src/components/Modal/index.jsx

```diff
@@ -0,0 +1,113 @@
+      setCompany(data.company)
+      setType(data.type != null ? data.type : 'N/A')
+      setImg(data.logo)
+      setDesc(data.job_description)
+      setSalary(data.salary != null ? data.salary : ' -')
+      setDate(data.posted)
+      setSkills(data.skills)
+    })
+  }
+  // console.log(jobTitle)
+  // left-1/2 top-1/2 transform -translate-x-1/2 -translate-y-1/2
+
+  return (
+    <section onClick={toggleModal} className='p-2.5 z-10 bg-zinc-500/50 w-full min-h-screen fixed overflow-auto top-0 left-0'>
+      <div className='Modal-container px-4 m-2 md:mx-auto md:my-2 w-full md:max-w-screen-md overflow-auto z-20 absolute my-5 bg-white left-1/2 transform -translate-x-1/2'>
+        <div className='flex justify-between my-4 ml-2 border-slate-200 border-b-2 pb-4'>
+          <div className='flex flex-row gap-2 font-bold'>
+            <h2>{`${jobTitle} -`}</h2>
+            <h2>{jobCompany}</h2>
+          </div>
+          <span>
+            <svg onClick={toggleModal} xmlns='http://www.w3.org/2000/svg' fill='none' viewBox='0 0 24 24' strokeWidth='3' stroke='currentColor' className='w-6 h-6 stroke-blue-500'>
+            <path strokeLinecap='round' strokeLinejoin='round' d='M6 18 18 6M6 6l12 12' />
+            </svg>
+          </span>
+        </div>
+        <div className='bg-zinc-50'>
+          <div className='logo-company text-white bg-blue-500 flex flex-row gap-3 items-center pl-5 py-10'>
+            <h1 className='font-semibold text-lg'>{jobTitle}</h1>
+            <h2 className='bg-green-600 rounded-full px-1.5 py-0.5 font-bold'>{jobType}</h2>
+          </div>
+          <div className='bg-white m-4 p-1'>
+            <div className='key-details m-5'>
+              <div className='flex flex-row gap-2 items-center'>
+                <img src={jobImg} alt='company logo' width={65} />
+                <h2 className='font-semibold'>{jobCompany}</h2>
+              </div>
+              <div className='font-semibold'>Key facts: </div>
+                <div className='facts flex flex-row gap-12'>
+                  <div className='flex flex-col gap-1'>
+                    <div>Salary:</div>
+                    <div>{`£${jobSalary}`}</div>
+                  </div>
+                  <div className='flex flex-col gap-1'>
+                    <div>Type: </div>
+                    <div>{jobType}</div>
+                  </div>
+                  <div className='flex flex-col gap-1'>
+                    <div>Data posted: </div>
+                    <div>{jobDate}</div>
+                  </div>
+                  <div className='flex flex-col gap-1'>
+                    <div>Skills</div>
+                      <div className='flex flex-row gap-1'>
+                        {jobSkills.map((skill) => (
+                        <span key={skill.id} className='bg-blue-500 text-white p-1 rounded-lg'>
+                          {skill.skill}
+                        </span>
+                      ))}
+                      </div>
```

Comment:
There's a small isssue here with responsivity, on a very small screen the skills get cut off